cirq/ops/common_gates.py

```diff
@@ -275,22 +275,30 @@ class RotZGate(eigen_gate.EigenGate,
     def __init__(self, *,  # Forces keyword args.
                  half_turns: Optional[Union[value.Symbol, float]] = None,
                  rads: Optional[float] = None,
-                 degs: Optional[float] = None) -> None:
+                 degs: Optional[float] = None,
+                 global_shift_in_half_turns: float = 0.0) -> None:
         """Initializes the gate.
 
-        At most one angle argument may be specified. If more are specified,
-        the result is considered ambiguous and an error is thrown. If no angle
-        argument is given, the default value of one half turn is used.
+        At most one relative phasing angle argument may be specified. If more
+        are specified, the result is considered ambiguous and an error is
+        thrown. If no angle argument is given, the default value of one half
+        turn is used.
 
         Args:
             half_turns: The relative phasing of Z's eigenstates, in half_turns.
             rads: The relative phasing of Z's eigenstates, in radians.
             degs: The relative phasing of Z's eigenstates, in degrees.
+            global_shift_in_half_turns: Offsets the eigenvalues of the gate.
+                The default shift of 0 gives the Z gate's matrix eigenvalues of
+                +1 and -1, whereas a shift of -0.5 changes those eigenvalues to
+                i and -i. The shift is always specified assuming an exponent of
```

Comment:
It would be clearer to say `-i and i` instead of `i and -i` so that the order matches with `+1 and -1`.

Reply:
Done.